site/app/views/AuthenticationView.php

```diff
@@ -7,6 +7,9 @@ public function loginForm() {
         $return = <<<HTML
 <div class="content">
     <h2>Submitty Login</h2>
+    <!--
+    <em>Note: Registered students have not yet been uploaded to the Submitty course databases.  Please check back later.</em>
+    -->
```

Comment:
Can we just remove this line completely and the next time this is needed implement some version of #1415?

Reply:
I'm leaving this in here until we have the feature because I may need to message everyone again.